Powershell-PoC-Environment-Setup.ps1

```diff
@@ -20,12 +20,81 @@
+19 OCT 2016 0.00.00.0009 Preston K. Parsard prestopa@microsoft.com Added $BeginTimer variable at the start of the script so that total script execution time can be measured at script completion.
+19 OCT 2016 0.00.00.0010 Preston K. Parsard prestopa@microsoft.com Construct custom path for log files based on current user's $env:HOMEPATH directory for both the log and transcript files.
+19 OCT 2016 0.00.00.0011 Preston K. Parsard prestopa@microsoft.com Create both log and transcript files with time/date stamps included in their filenames.
+19 OCT 2016 0.00.00.0012 Preston K. Parsard prestopa@microsoft.com Added work-items (tasks) comment section to track outstanding tasks.
+19 OCT 2016 0.00.00.0013 Preston K. Parsard prestopa@microsoft.com Added region tags to accomodate collapsing sections of script to hide details or make it easier to scroll.
+19 OCT 2016 0.00.00.0014 Preston K. Parsard prestopa@microsoft.com Create prompt and responses custom object for opening logs after script completes.
+19 OCT 2016 0.00.00.0015 Preston K. Parsard prestopa@microsoft.com Add logging module: WriteToLogs.
+19 OCT 2016 0.00.00.0016 Preston K. Parsard prestopa@microsoft.com Add and display header.
+19 OCT 2016 0.00.00.0017 Preston K. Parsard prestopa@microsoft.com Format and truncate the results of the New-Guid cmdlet for a subset of random numeric and lowercase combination of characters
+19 OCT 2016 0.00.00.0018 Preston K. Parsard prestopa@microsoft.com Add a random infix (4 numeric digits) inside the Dnslabel name to avoid conflicts with existing deployments generated from this script. 
+19 OCT 2016 0.00.00.0019 Preston K. Parsard prestopa@microsoft.com Create a new random string, then extract the 4 digits to use as the last characters for the storage account name for each region.
+19 OCT 2016 0.00.00.0020 Preston K. Parsard prestopa@microsoft.com Generate a pseudo-random password based on the prefix "SAFE" to satisfy the uppercase characters complexity requirement, plus a random 
+...................................................................combination of 8 lowercase and numeric characters. As such, this meets the password complexity requirement of 3 of the 4 complexity rules,
+...................................................................while reducing the probability that an offensive word will be generated.
+19 OCT 2016 0.00.00.0021 Preston K. Parsard prestopa@microsoft.com Use previously captured plain-text password variable instead of hard-coding in script.
+19 OCT 2016 0.00.00.0021 Preston K. Parsard prestopa@microsoft.com Added footer region to calculate elapsed time, display footer message, prompt to open log and transcript files, stop transcript...
+...................................................................as well as added a commented section that can be used to decomission PoC environment for test/dev situations in order to clean up resources & reduce cost
+#>

```

Comment:
these lines from 83 and 87 are not needed, GitHub is being our source control and versioning system, no need to add here, lets keep it simple.